client/src/views/header/NotificationCenterPopover.vue

```diff
@@ -82,19 +81,140 @@ onMounted(() => {
         },
       });
       informationManager.present(info, PresentationMode.Notification);
-    }, i * 5000);
+
+      const info1 = new Information({
+        message: `Notification ${i}`,
+        level: InformationLevel.Info,
+        data: {
+          type: InformationDataType.NewWorkspaceAccess,
+          workspaceId: '1337',
+          role: WorkspaceRole.Manager,
+        },
+      });
+      informationManager.present(info1, PresentationMode.Notification);
+
+      const info2 = new Information({
+        message: `Notification ${i}`,
+        level: InformationLevel.Info,
+        data: {
+          type: InformationDataType.NewWorkspaceAccess,
+          workspaceId: '1337',
+          role: WorkspaceRole.Manager,
+        },
+      });
+      informationManager.present(info2, PresentationMode.Notification);
+
+      const info3 = new Information({
+        message: `Notification ${i}`,
+        level: InformationLevel.Info,
+        data: {
+          type: InformationDataType.UserJoinWorkspace,
+          workspaceId: '1337',
+          role: WorkspaceRole.Manager,
+          user: {
```

Comment:
No, a user will by represented by their id (a `UserID`). But the notification should retrieve the HumanHandle using this id.